app/models/spree/order.rb

```diff
@@ -47,6 +47,7 @@ class Order < ActiveRecord::Base
              dependent: :destroy
 
     has_many :line_item_adjustments, through: :line_items, source: :adjustments
+    has_many :all_adjustments, class_name: 'Spree::Adjustment', dependent: :destroy
```

Comment:
I was wondering how this worked. Printing the sql reminded me that we recently added `order_id` to adjustments to make this possible.

Reply:
Yeah it's really useful. It's all the combined adjustments on the order, it's line items, and it's payments and shipments, and it allows things like:
```ruby
additional_tax_total = order.all_adjustments.tax.additional.sum(:amount)
```

This is awesome :point_up: